cortx-motr.spec.in

```diff
@@ -201,10 +219,14 @@ CORTX Motr IVT.
 %setup -q
 
 %build
-%configure %{with_linux} %{with_lustre} %{configure_opts}
-%if !%{with ivt}
-make %{?_smp_mflags}
+%if  "%{CONFIG_ARCH_TYPE}"=="x86_64"
```

Comment:
s/  / / (change double-space to single-space)

Reply:
Fixed